### PR TITLE
[Feat]combine cpu and gpu wheel

### PR DESCRIPTION
**Pull request by jq**
# Description

Brief Description of the PR:
combine cpu and gpu wheel so that you only need pip install tensorflow-recommenders-addons
Fixes # (issue)

## Type of change

- [ ] Bug fix
- [ ] New Tutorial
- [ ] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Feature

# Checklist:

- [ ] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/recommenders-addons/blob/master/CONTRIBUTING.md#coding-style)
    - [ ] By running yapf
    - [ ] By running clang-format
- [ ] This PR addresses an already submitted issue for TensorFlow Recommenders-Addons
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ run the following:
 pip install tensorflow-recommenders-addons
 ```
 
-By default, CPU version will be installed. To install GPU version, run the following:
+Before the release of combined CPU and GPU version, to install GPU version, run the following:
```
Comment (rhdong) [inline README.md:79]: Better to explain more clearly, for example, the specific version(0.7.x) after which we make this change.

**Review by rhdong — DISMISSED** [review]
LGTM

**Comment by rhdong** [comment]
Hey @MoFHeka pls review it when you feel convenient, basically, it's LGTM.

File: README.md (hunk 1 above)
Comment (jq) [reply README.md]: updated

**Review by MoFHeka — APPROVED** [review]
LGTM

**Pull request merged**